daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/SeparatedSequenceUnparsers.scala

```diff
@@ -155,7 +165,7 @@ class OrderedSeparatedSequenceUnparser(
     state: UState,
     trailingSuspendedOps: Buffer[SuppressableSeparatorUnparserSuspendableOperation]): Unit = {
 
-    val suspendableOp = new SuppressableSeparatorUnparserSuspendableOperation(sep, trd)
+    val suspendableOp = new SuppressableSeparatorUnparserSuspendableOperation(sepMtaAlignmentMaybe, sep, trd)
```

Comment:
I'm seeing a coverage warning on this line also. Definitely need to see this exercised to be sure the SupressableSeparatorUnparserSuspendableOperation is being tested.  Not sure why it wouldn't have been.

Reply:
Yeah, I'm pretty sure this line is hit, otherwise some tests would fail. I'll look into what's going on.

Reply:
I have confirmed that this is in fact not hit by any tests (these new codecov warnings are nice!). The new logic for alignment is tested because the ``unparseOneWithSupression`` function also creates this same suspension.

After grepping, this ``unparseJustSeparatorWithTrailingSuppression`` function is actually just never called anywhere. I suspect this is just dead code and ``unparseOneWithSuppression`` handles all cases of suppressed separators, including the trailing suppression case? Any idea of this can just be removed, or if we're perhaps missing logic somewhere else and this function should be used?

Reply:
Yow. Well, the purpose of this must be for unparsing an empty element that is not trailing, so not trailing suppressed. There are some bugs about separatorSuppressionPolicy="never" which this code might be in service of. DAFFODIL-2219 is one. 

Put a nice big comment in here with a FIXME: possibly dead code, but maybe needed to fix separatorSuppressionPolicy bugs like DAFFODIL-2219

Reply:
Probably create a JIRA ticket about this dead code as well.

Reply:
To avoid the coverage thing, I suppose this code must be commented out. I'm just reluctant to delete it given the open bugs in this area.

Reply:
Fair, I'll comment it out and create an issue + comment so we don't lose this if it's needed.

Reply:
Fixed in latest fixup commit